lib/provider-manager.js

```diff
@@ -1,21 +1,19 @@
-const pkg = require('../package.json');
-const globs = require('./globs');
+import * as globs from './globs.js';
+import pkg from './pkg.cjs';
 
-const levels = {
+export const levels = {
```

Comment:
IMHO, if the only reasonably way to import a package is `import * as foo`, then I think it should just be an object with a default export instead of single exports.